examples/website/data-filter/wrapper.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
```

Comment:
Following React naming convention, I'd call

- the file root.js
- `function Root({data})`
- `render(<Root data={data} />, container)`